packages/gluestick/src/config/webpack/deepCopy.js

```diff
@@ -0,0 +1,14 @@
+const deepCopy = (obj: Object) => {
```

Comment:
Wouldn't be better to use existing implementation ? Like `cloneDeep` from lodash? Then we do not need to re-do it and re-test it.